examples/004_non_linear_fit/x_errors.yml

```diff
@@ -34,3 +34,6 @@ y_data:
 - 12.126133797209802
 - 15.805993018808897
 - 15.3488445186788
+# The x errors are much bigger than the y errors. This will cause a distortion of the likelihood function.
```

Comment:
The second sentence makes it sound like distortions only happen if x errors >> y errors.
I suggest the following wording for the second sentence:

This will strongly distort the likelihood function.